app/src/main/java/com/example/bigowlapp/activity/ScheduleTestActivity.java

```diff
@@ -0,0 +1,75 @@
+package com.example.bigowlapp.activity;
+
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.widget.Button;
+
+import com.example.bigowlapp.R;
+
+import java.util.Random;
+
+import androidx.annotation.RequiresApi;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
+
+// TODO remove the whole activity
+public class ScheduleTestActivity extends AppCompatActivity {
+    final String channelId = "ScheduleNotificationChannelId";
+    NotificationCompat.Builder builder;
+    int nid;
```

Comment:
All three variables should have an access modifier of private.

More importantly, these variables are not descriptive enough; please change their names for more descriptive ones.